python/ray/actor.py

```diff
@@ -498,6 +499,11 @@ def _remote(self,
         else:
             detached = False
 
+        if placement_group_id is not None and bundle_index is None:
+            raise ValueError("The placement_group_id is set."
+                             "But the bundle_index is not set.")
```

Comment:
Can we allow None to mean any bundle?

Reply:
You mean random a boundle_index if the it is None?

Reply:
@wumuzi520, I discussed with @ericl before, supporting any bundle is a valid use case.

Reply:
I'm not sure whether the backend already supports this though.

Reply:
Ok, maybe we can add it later.